web/cofig.php

```diff
@@ -0,0 +1,113 @@
+<?php
```

Comment:
I think this whole file was added accidentally -- you should never commit your local config.php because it will overwrite everyone else's local config

Reply:
Yes this was a mistake. Sorry about that. I'll fix this.